autoload/beancount.vim

```diff
@@ -88,45 +171,45 @@ function! beancount#complete_account(findstart, base)
     return l:matches
 endfunction
 
-" Get list of acounts.
-function! beancount#find_accounts(root_file)
-    python << EOM
-import collections
-import os
-import re
-import sys
+function! beancount#query_single(root_file, query)
+python << EOF
 import vim
+import subprocess
+import os
+
+# We intentionally want to ignore stderr so it doesn't mess up our query processing
+tagoutput = subprocess.check_output(['bean-query', vim.eval('a:root_file'), vim.eval('a:query')], stderr=open(os.devnull, 'w')).split('\n')
+tagoutput = tagoutput[3:]
```

Comment:
Not a good reason, but a lot of the output ends up having another blank line (select distinct tags, for example).  I fixed the general problem in the list comprehension by stripping and dropping the empty ones.  Only removing 2 lines now.